redisq/queue.go

```diff
@@ -330,14 +330,14 @@ func (q *Queue) scheduleDelayed(ctx context.Context) (int, error) {
 }
 
 func (q *Queue) schedulePending(ctx context.Context) (int, error) {
```

Comment:
Let's add a brief comment what this method does? There is a good explanation in https://github.com/vmihailenco/taskq/issues/115 and perhaps linking https://docs.aws.amazon.com/AWSSimpleQueueService/latest/SQSDeveloperGuide/sqs-visibility-timeout.html is a good thing too.